### PR TITLE
just a few small clarifications

### PR DESCRIPTION
**Pull request by osmiumic**

### CODE REVIEW

File: documentation/dsl.md [hunk 1]
```diff
@@ -169,7 +169,7 @@ On the other hand fractions like `9/8` are in the *linear* domain. The musically
...
 Relative FJS intervals can be especially confusing because they represent fractions but add like cents do. Our whole-tone stack becomes `M2 + M2` equal to `M3`. This is notationally expected even if obscured by the ordinal nature of traditional Western music notation. Remember that the perfect unison `P1` represents no change and `P1 + P1` is equal to `P1` i.e. 1st + 1st = 1st. Therefore 2nd + 2nd must be a (2+2-1)rd = 3rd.
 
-For the most part you cannot combine intervals across domains so `9/8 + M2` is not a valid operation. Use tildes (`~`) to always operate as if in the linear domain. E.g. `9/8 ~* M2` is a valid expression and evaluates to `81/64` while `9/8 *~ M2` evaluates to `M3`. The direction of the "wing" determines which domain and formatting to prefer.
+For the most part you cannot combine intervals across domains so `9/8 + M2` is not a valid operation. Use tildes (`~`) to always operate as if in the linear domain (meaning that, for example, `+` and `*` will always work as they are in maths). E.g. `9/8 ~* M2` is a valid expression and evaluates to `81/64` while `9/8 *~ M2` evaluates to `M3`. The direction of the "wing" determines which domain and formatting to prefer.
```
Comment (frostburn) [inline documentation/dsl.md:172]: "work as they **do**", right?

Comment (osmiumic) [reply documentation/dsl.md:172]: sorry yes. in fact you can remove "they are" / "they do" entirely; "as in maths" is clearer (that's the original wording i used iirc)

File: documentation/dsl.md [hunk 2]
```diff
@@ -214,7 +214,7 @@ SonicWeave doesn't have percentages so the CSS color `hsl(120, 60%, 70%)` is spe
 
 ## Code comments
 
-Anything after two slashes (`//`) is ignored until the end of the line. Everything after a slash and an asterisk (`/*`) is ignored until a corresponding pair (`*/`) is encountered.
+Anything after two slashes (`//`) is ignored until the end of the line. Everything after a slash and an asterisk (`/*`) is ignored until a corresponding pair (`*/`) is encountered. (This means you cannot nest multi-line comments, as in C and javascript.)
```
Comment (frostburn) [inline documentation/dsl.md:217]: I think I've capitalized it *JavaScript* in the docs. Would be nice to have consistency. (Lest we say *ECMAScript* in order to be accurate but obscure.)

Comment (osmiumic) [reply documentation/dsl.md:217]: ah right. ive usually seen javascript in all lowercase when mentioned formally, as suggested by the abbreviation "js".

**Comment by frostburn** [comment]
Good additions! :+1:
I'll squash everything into one commit after the minor fixes are done unless you've already squashed on your end.

**Review by frostburn — APPROVED** [review]

**Pull request merged**